### ISSUE TITLE
[Tutorials] Update tutorials to use cssClass instead of glyph

### ISSUE THREAD

**Issue by crgimenes** [nasa/openmct#1163]
I'm following the tutorial to create a todolist in https://nasa.github.io/openmct/docs/tutorials/#step-2-add-a-domain-object-type,

I followed the steps carefully one by one and the new item on the menu create is showing.
But the icon is not like in the tutorial, in place of this icon appears only a question mark.

I do not know if this related but in the browser I am getting the following errors:

GET http://localhost:8080/platform/commonUI/general/res/css/startup-base.css net::ERR_CONNECTION_REFUSED
openmct.css:1 GET http://localhost:8080/platform/commonUI/general/res/css/openmct.css net::ERR_CONNECTION_REFUSED
normalize.min.css:1 GET http://localhost:8080/platform/commonUI/general/res/css/normalize.min.css net::ERR_CONNECTION_REFUSED
table.css:1 GET http://localhost:8080/platform/features/table/res/css/table.css net::ERR_CONNECTION_REFUSED
theme-espresso.css:1 GET http://localhost:8080/platform/commonUI/themes/espresso/res/css/theme-espresso.css net::ERR_CONNECTION_REFUSED
timeline.css:1 GET http://localhost:8080/platform/features/timeline/res/css/timeline.css net::ERR_CONNECTION_REFUSED
timeline-espresso.css:1 GET http://localhost:8080/platform/features/timeline/res/css/timeline-espresso.css net::ERR_CONNECTION_REFUSED

This is the code that is coming to the new item todolist:

`
<a ng-mouseover="representation.activeMetadata = createAction.getMetadata()" ng-mouseleave="representation.activeMetadata = undefined" class="menu-item-a icon-object-unknown">
                    To-Do List
                </a>
`

It is not a big problem but visually draws much attention.
I would also like to know where to find a list of possible values for the parameter "glyph"

**Comment by hudsonfoo** [comment]
@crgimenes I *think* they are switching from `glyph` to `cssclass` in the `bundle.js` and just haven't updated the tutorials yet. An example might be:

```diff
define([
    'legacyRegistry'
], function (
    legacyRegistry
) {
    legacyRegistry.register("tutorials/todo", {
        "name": "To-do Plugin",
        "description": "Allows creating and editing to-do lists.",
        "extensions":
        {
         "types": [
          {
              "key": "example.todo",
              "name": "To-Do List",
+             "cssclass": "icon-pencil",
              "description": "A list of things that need to be done.",
              "features": ["creation"]
          }
       ]}
    });
});

**Comment by crgimenes** [comment]
You're right @hudsonfoo. The tutorial is outdated and I'm still learning how the system works. I gave up to follow the tutorial and I am trying the examples.
Thank you for your help!

**Comment by akhenry** [comment]
Ah, good catch. Yes, the tutorials need to be updated. @hudsonfoo is correct, we are switching from glyphs to css classes in our code. The correct css class for this tutorial is, I believe, `icon-check`.

The list of available css classes is not currently published, but will be as part of an upcoming public style guide.

The tutorials should be updated to refactor out the old glyphs and replace them with CSS classes.

**Comment by VWoeltjen** [comment]
Putting this in Rajaniemi, which should coincide reasonably well with the [new API](https://github.com/nasa/openmct#new-api) (at which point the tutorials will need to be largely rewritten.)

**Comment by crgimenes** [comment]
I would like to suggest to the new API is covered by unit-tests. Test would have helped a lot to learn how things work even without the documentation.
How to get involved in developing the new API? There is already some material to read?

**Comment by VWoeltjen** [comment]
> I would like to suggest to the new API is covered by unit-tests.

👍 I think this is a good idea; in addition to the reason you mention, I'd like to have tests in place to detect breaking changes to the public API.

The discussions around the API are somewhat fragmented; #69 and #999 have a lot of background, whereas #1110, #1111 and #1124 are more current/upcoming tasks. We definitely welcome any input you might have!

**Comment by crgimenes** [comment]
Hello @VWoeltjen. 
How about we use the wiki, that here in github, to document the draft of this new API and centralize the discussion?

**Comment by temecom** [comment]
I have been trying to get the To Do tutorial to work and cannot get it to register the 'Create/Todo List'  menu item. I created the bundle.js and added it to defaultRegistry.js - no joy (no errors either). 

What is the secret sauce?

**Comment by temecom** [comment]
Never mind - Need to add it to the index.html script: 
<script>
        require(['openmct'], function (openmct) {
            [
                'example/imagery',
                'example/eventGenerator',
                'example/generator',
                'tutorials/todo',
                'platform/features/my-items'
            ].forEach(
                openmct.legacyRegistry.enable.bind(openmct.legacyRegistry)
            );
            openmct.start();
        });
    </script>

**Comment by akhenry** [comment]
This issue has been resolved by #1339
